protobuf/coreapi/api.proto

```diff
@@ -42,6 +42,9 @@ service Core {
 
   // ServiceLogs gives a stream for dependency logs of a service.
   rpc ServiceLogs (ServiceLogsRequest) returns (stream LogData) {}
+
+  // Info returns all necessary informations from the core.
+  rpc Info(InfoRequest) returns (InfoReply) {}
```

Comment:
s/Info/CoreInfo/ but i'm not sure about this

Reply:
I think Info is fine, we are accessing the API of the Core already